crates/voicevox_core/src/error.rs

```diff
@@ -82,6 +82,27 @@ pub enum Error {
 
     #[error("{},{0}", base_error_message(VOICEVOX_RESULT_PARSE_KANA_ERROR))]
     ParseKana(#[from] KanaParseError),
+
+    #[error(
+        "{}: {0}",
+        base_error_message(VOICEVOX_OPEN_JTALK_LOAD_USER_DICT_ERROR)
+    )]
+    UserDictLoad(String),
+
+    #[error("{}: {0}", base_error_message(VOICEVOX_USER_DICT_SAVE_ERROR))]
+    UserDictSave(String),
+
+    #[error("{}: {0}", base_error_message(VOICEVOX_USER_DICT_WORD_NOT_FOUND_ERROR))]
+    WordNotFound(String),
```

Comment:
ここに入るのは`Uuid`なので、`Uuid`をそのまま持てばいいと思います。

```suggestion
    WordNotFound(Uuid),
```